extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/operations/KuduMetadataDeleter.java

```diff
@@ -50,5 +60,23 @@ public boolean delete(final MetadataQuery query) {
   }
 
   @Override
-  public void flush() {}
+  public void flush() {
+    try {
+      session.flush();
+      if (session.countPendingErrors() != 0) {
+        LOGGER.error(
+            "Got {} pending errors while flushing Kudu session",
+            session.countPendingErrors());
+      }
+    } catch (KuduException e) {
+      LOGGER.error("Encountered error while flushing Kudu session", e);
+    }
+  }
+
+  @Override
+  public void close() throws Exception {
```

Comment:
Why does the metadata deleter throw and exception while the row deleter does not?

Reply:
The interface overrides close to throw Exception